test/e2e/data/kubetest/conformance.yaml

```diff
@@ -0,0 +1,8 @@
+ginkgo.focus: \[Conformance\]
+ginkgo.skip: Alpha|\[(Disruptive|Feature:[^\]]+|Flaky)\]
```

Comment:
@randomvariable this is slightly different than the one used by CAPA, but it is conformant with the instruction I have found https://github.com/kubernetes/test-infra/tree/master/testgrid/conformance.
Should we converge?

Reply:
That info is out of date and should be updated in test-infra. The disruptive tests were removed from the conformance suite so it's no longer necessary.

Reply:
Gonna PR that now in fact.

Reply:
https://github.com/kubernetes/test-infra/pull/19604

Reply:
Thanks for the context, I will align to CAPA settings then

Reply:
Were you going to update this one?

Reply:
done!